app/code/Magento/ProductAlert/Controller/Add/Price.php

```diff
@@ -6,16 +6,18 @@
 
 namespace Magento\ProductAlert\Controller\Add;
 
-use Magento\Framework\App\Action\HttpGetActionInterface;
-use Magento\ProductAlert\Controller\Add as AddController;
-use Magento\Framework\App\Action\Context;
-use Magento\Customer\Model\Session as CustomerSession;
-use Magento\Store\Model\StoreManagerInterface;
 use Magento\Catalog\Api\ProductRepositoryInterface;
-use Magento\Framework\UrlInterface;
+use Magento\Customer\Model\Session as CustomerSession;
 use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\App\Action\HttpGetActionInterface;
 use Magento\Framework\Controller\ResultFactory;
+use Magento\Framework\Controller\Result\Redirect;
 use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\UrlInterface;
+use Magento\ProductAlert\Controller\Add as AddController;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Tests\NamingConvention\true\string;
```

Comment:
This is mistake, please remove it.

Note: it's better to ignore `vendor/magento/magento-coding-standard` folder in PhpStorm in order not to have such mistakes

Reply:
**done**